.github/workflows/GuideConverterAction.yaml

```
@@ -0,0 +1,100 @@
+name: GuideConverter
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch pushed to'     
+        required: true
+        default: 'defaulted'
+      guide_name:
+        description: 'Guide changed'
+        required: true
+        default: 'not given'
+        
+jobs:
+  printBranch:
+    runs-on: ubuntu-latest
+
+    steps: 
+
+      - name: Output message
+        run: |
+          echo "${{ github.event.inputs.branch }}"
+          echo "${{ github.event.inputs.guide_name }}"
```

```suggestion
          echo "${{ github.event.inputs.guide_name }}"
          
```

Add line between jobs.

---

Done